tsconfig.json

```diff
@@ -1,18 +1,25 @@
 {
   "compilerOptions": {
-    "target": "es5",
-    "module": "esnext",
+    "target": "es2019",
+    "module": "ESNext",
     "lib": ["dom", "esnext"],
     "declaration": true,
     "sourceMap": true,
-    "moduleResolution": "node10",
-    "strict": true,
-    "isolatedModules": true,
-    "noFallthroughCasesInSwitch": true,
-    "jsx": "react-jsx",
     "esModuleInterop": true,
     "allowSyntheticDefaultImports": true,
     "downlevelIteration": true,
+    "skipLibCheck": true,
```

Comment:
It was necessary to add this in order to prevent Chakra UI from causing the `tsc` to fail. Not an ideal solution.

- https://github.com/chakra-ui/chakra-ui/issues/7203
- https://github.com/chakra-ui/chakra-ui/issues/7116